editor/src/components/inspector/sections/style-section/text-subsection/font-family-select-popup.tsx

```diff
@@ -496,7 +504,7 @@ export const FontFamilySelectPopup = betterReactMemo(
           tabIndex={0}
           style={{
             ...style,
-            backgroundColor: 'white',
+            backgroundColor: colorTheme.inspectorBackground.value,
```

Comment:
This is inside a functional component so you could use the `useColorTheme` hook to get the color theme, which I believe is what you intended to do here but perhaps accidentally imported `colorTheme` (via autocomplete?) instead

Reply:
Yepp. Fixed